client/me/purchases/payment/edit-card-details/index.jsx

```diff
@@ -92,7 +92,7 @@ function EditCardDetails( props ) {
 						configurationArgs={ { needs_intent: true } }
 						fetchStripeConfiguration={ getStripeConfiguration }
 					>
-						<CreditCardForm
+						<PaymentMethodForm
```

Comment:
I'm not sure I follow why the previous `EditCardDetails` component is getting the new `PaymentMethodForm` and the new `ChangePaymentMethod` is using the previous `CreditCardForm`. Could you help explain? Thanks.